fixtures.sh

```diff
@@ -63,3 +63,9 @@ docker run -d --name runlike_fixture5 \
     --link runlike_fixture4:alias_of4 \
     --link runlike_fixture1 \
     runlike_fixture
+
+docker run -d --name runlike_fixture6 \
```

Comment:
👍 I probably should have added `--rm` to all of them. Good idea.

Reply:
However, I'm trying to keep the number of container fixtures as low as possible. Otherwise, by the feature count, we would have had dozens of them by now in the test suite. I think you could just add the entrypoint to fixture 2 without impacting anything preexisting.